core/src/components/datetime/datetime.md.scss

```diff
@@ -54,7 +54,7 @@
 // Calendar / Body
 // -----------------------------------
 :host .calendar-body .calendar-month .calendar-month-grid {
-  @include padding(0px, 10px, 0px, 10px);
+  @include padding(3px, 10px, 0px, 10px);
```

Comment:
I think this could easily be regressed by someone who is not aware of why this has to be 3px. Can we add a comment explaining why this is necessary?